### PR TITLE
Issue 437: Maintenance permission

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by spong** [comment]
Referenced images will need to be updated as well:


<p align="center">
  
</p>

**Comment by spong** [comment]
So the intent behind https://github.com/elastic/kibana/pull/89147 was to allow the creation of a user (role) that can change alert state (open/in-progress/closed), but not have the ability to create/edit rules. Can we create a third group like `Access and use Detections` for this role and call out this behavior? It'd essentially be what the `Access and use Detections` section was previously (no `maintenance`) and a change of Kibana privileges to Security Solution `read`, if I recall correctly.

**Comment by DonNateR** [comment]
[Preview](https://security-docs_492.docs-preview.app.elstc.co/guide/en/security/master/detections-permissions-section.html)

**Comment by spong** [comment]
There's a bit to un-pack here @DonNateR @jmikell821, so please do reach out as I'm happy to pair on this one.

Notes: 

[1] As of https://github.com/elastic/kibana/pull/90895 the `Saved Objects Management` feature privilege is no longer necessary, and can be removed from both sections.

[2] For the _Enable Detections_ section `maintenance` isn't necessary as the `manage` privilege will satisfy the same permissions.

<p align="center">
  
</p>

**Comment by DonNateR** [comment]
[Preveiw](https://security-docs_492.docs-preview.app.elstc.co/guide/en/security/master/detections-permissions-section.html)

File: docs/getting-started/detections-req.asciidoc [hunk 1]
```diff
@@ -42,11 +42,10 @@ and restarting {kib}, you must restart all detection rules.
 To enable the <<detection-engine-overview, Detections feature>>, a user with
 these privileges must visit (click on) the *Detections* page:
 
-* The `manage` cluster privilege
+* The `manage` cluster privilege.
 * {kib} space `All` privileges for the `Security` feature (see
 {kibana-ref}/xpack-spaces.html#spaces-control-user-access[Feature access based on user privileges]).
 * The `manage`, `write`,`read`, and `view_index_metadata` index privileges for all of these system indices:
-** `.siem-signals-<kib-space>`
```
Comment (spong) [inline docs/getting-started/detections-req.asciidoc:49]: I think this is an accidental delete? We'll still need this along with `lists`/`items` right below.

File: docs/getting-started/detections-req.asciidoc [hunk 2]
```diff
@@ -74,24 +73,32 @@ image::images/sec-admin-user.png[]
...
-* {kib} space `All` privileges for the `Security`  and `Saved Objects
-Management` features
+**All**
+
+These permissions are required for both rule and alert management:
+
+* {kib} space with `All` privileges enabled for `Security`.
+* The `maintenance` permission for `.siem-signals-<kib-space>`.
 * The `read`, `write`, and `view_index_metadata` index privileges for all of these system indices:
-** `.siem-signals-<kib-space>`
```
Comment (spong) [inline docs/getting-started/detections-req.asciidoc:83]: We'll want this back as well

File: docs/getting-started/detections-req.asciidoc [hunk 3]
```diff
@@ -74,24 +73,32 @@ image::images/sec-admin-user.png[]
...
 
-* {kib} space `All` privileges for the `Security`  and `Saved Objects
-Management` features
+**All**
+
+These permissions are required for both rule and alert management:
+
+* {kib} space with `All` privileges enabled for `Security`.
+* The `maintenance` permission for `.siem-signals-<kib-space>`.
 * The `read`, `write`, and `view_index_metadata` index privileges for all of these system indices:
```
Comment (spong) [inline docs/getting-started/detections-req.asciidoc:84]:
```suggestion
* The `read`, `write`, `view_index_metadata`, and `maintenance` index privileges for all of these system indices:
```

Comment (spong) [reply docs/getting-started/detections-req.asciidoc:84]: This should match the image, where all three indices get the described privileges.

File: docs/getting-started/detections-req.asciidoc [hunk 4]
```diff
@@ -74,24 +73,32 @@ image::images/sec-admin-user.png[]
...
-*Detections* page:
+After enabling Detections, only users with these permissions can view and use rules and alerts on the *Detections* page:
 
-* {kib} space `All` privileges for the `Security`  and `Saved Objects
-Management` features
+**All**
+
+These permissions are required for both rule and alert management:
+
+* {kib} space with `All` privileges enabled for `Security`.
```
Comment (spong) [inline docs/getting-started/detections-req.asciidoc:82]: In `7.12` we can probably remove this in favor of calling it out below in the `**Rule**` and `**Alert**` sections. Rule will stay as `All`, and `Alerts` will be `Read`.

Comment (DonNateR) [reply docs/getting-started/detections-req.asciidoc:82]: I'll make a separate PR for this comment, once I've backported this one.

**Review by spong — APPROVED** [review]
LGTM! Thanks for pairing on this one to get everything just right @DonNateR, appreciate it! 🙂

**Pull request merged**